docs/lib/graphqlapi/concepts.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Concepts
+description: description
+---
+
+## AWS AppSync
+
+The Amplify Framework uses AWS AppSync, a managed service that uses GraphQL to make it easy for applications to get exactly the data they need. With AppSync, you can build scalable applications, including those requiring real-time updates, on a range of data sources such as NoSQL data stores, relational databases, HTTP APIs, and your custom data sources with AWS Lambda.
+
+For mobile and web apps, AppSync additionally provides SDKs that support local data access when devices go offline, and data synchronization with customizable conflict resolution, when they are back online.
+
+### The API Category
+
+The API category provides a solution for making HTTP requests to both GraphQL as well as REST endpoints. It includes a [AWS Signature Version 4](http://docs.aws.amazon.com/general/latest/gr/signature-version-4.html) signer class which automatically signs all AWS API requests for you as well as methods to use API Keys, Amazon Cognito User Pools, or 3rd party OIDC providers.
+
+The AWS Amplify API module supports AWS AppSync or any other GraphQL backends.
+
+<amplify-callout>
+
+To learn more about GraphQL, please visit the [GraphQL website](http://graphql.org/learn/).
+
+</amplify-callout>
+
+## Using AWS AppSync
+
+AWS AppSync helps you build data-driven apps with real-time and offline capabilities. Learn more about [AWS AppSync](https://aws.amazon.com/appsync/) by visiting [AWS AppSync Developer Guide](https://docs.aws.amazon.com/appsync/latest/devguide/welcome.html).
+
+The Amplify Framework offers three SDK options for AppSync.
+
+__[Amplify GraphQL client](~/lib/graphqlapi/query-data.md)__ - a light weight option if you're looking for a simple way to leverage GraphQL features and do not need the offline capabilities or caching. If you need those features, please look at [Amplify DataStore](https://aws-amplify.github.io/docs/js/datastore).
```

Comment:
Do we want to point to our new docs for DataStore?

Reply:
Good catch, just pushed this update.